cmd/create.go

```diff
@@ -145,6 +149,25 @@ func runCreate(cmd *cobra.Command, _ []string) error {
 				return fmt.Errorf("--organisation-id flag must be set")
 			}
 		}
+
+		if provider == "ec2" {
+			vpcID, err = cmd.Flags().GetString("vpc-id")
+			if err != nil {
+				return errors.Wrap(err, "failed to get 'vpc-id' value.")
+			}
+			if len(vpcID) == 0 {
+				return fmt.Errorf("--vpc-id flag must be set")
```

Comment:
there's no requirment for setting these, only if your account doesnt have default or you want to specify another. Can these not be errors if len(0)